-------------------------------------------------------------------------------
Test set: TestSuite
-------------------------------------------------------------------------------
Tests run: 2, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 7.841 s <<< FAILURE! - in TestSuite
placeOrder(tests.base.PlaceOrderTest)  Time elapsed: 1.159 s  <<< FAILURE!
java.lang.AssertionError: expected [THANK YOU FOR YOUR ORDER] but found [Thank you for your order!]
	at tests.base.PlaceOrderTest.placeOrder(PlaceOrderTest.java:40)

